sdk/identity/azure-identity/azure/identity/_authn_client.py

```diff
@@ -14,6 +15,12 @@
 from azure.core.pipeline.policies import ContentDecodePolicy, NetworkTraceLoggingPolicy, ProxyPolicy, RetryPolicy
 from azure.core.pipeline.policies.distributed_tracing import DistributedTracingPolicy
 from azure.core.pipeline.transport import RequestsTransport
+from azure.identity._constants import AZURE_CLI_CLIENT_ID
```

Comment:
FYI I made clientId a required parameter in the ctor for .NET and just use this when I construct it in the DefaultAzureCredential